presto-accumulo/pom.xml

```diff
@@ -368,5 +368,26 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>skipJdk17IncompatibleTests</id>
```

Comment:
We'll need a release note / action plan for deprecating Accumulo

Reply:
Let's bring it up as a topic at TSC?